### ISSUE TITLE
real device as a backing store

### ISSUE THREAD

**Issue by krinkinmu** [starpos/crashblk#1]
Memory backed block device is very limited in use for filesystem testing due to small size, so i would suggest to rewrite craschblk to use a block device as a backing store. And if you are interested i would like to help with this (i'm not very experienced kernel developer though).

**Comment by starpos** [comment]
Thank you for your comments.

I considered to make crashblk use backing store at the design phase and I decided to use memory devices. The primary reason is to set arbitrary response time to reproduce various performance behavior, not to be affected by limitation of real backing stores. The secondary reason is a result of a trade-off between simplicity and the importance of big-size test.

It may be nice to implement such function additionally, 
but I do not have any motivation to implement it now.
If you are interested in it, fork the repository and and hack it!

BTW, there is similar project dm-flakey (I did not know it).
It uses backing stores but its functions are almost different from crashblk now.
Another way is improve dm-flakey to make it have crashblk functions.
